src/app/Scenes/Activity/components/PartnerOfferCreatedNotification.tsx

```diff
@@ -16,12 +17,15 @@ interface PartnerOfferCreatedNotificationProps {
 export const PartnerOfferCreatedNotification: React.FC<PartnerOfferCreatedNotificationProps> = ({
   notification,
 }) => {
-  const notificationData = useFragment(PartnerOfferCreatedNotificationFragment, notification)
+  const notificationData = useFragment<PartnerOfferCreatedNotification_notification$key>(
+    PartnerOfferCreatedNotificationFragment,
+    notification
+  )
```

Comment:
Adding the type here shouldn't be necessary. You could instead set `notification`'s type in `PartnerOfferCreatedNotificationProps` to `PartnerOfferCreatedNotification_notification$key`. For some reason it got an `any` type.

```suggestion
  const notificationData = useFragment(
    PartnerOfferCreatedNotificationFragment,
    notification
  )
```